lib/internal/bootstrap_node.js

```diff
@@ -261,7 +261,8 @@
       enumerable: true,
       get: function() {
         if (!console) {
-          console = installInspectorConsole(originalConsole);
+          console = originalConsole ? installInspectorConsole(originalConsole) :
```

Comment:
Can the `originalConsole` check be stricter? Such as checking explicitly against `undefined`.